EiDiA_frontend/src/components/SmartDropDownBox.js

```diff
@@ -12,14 +12,31 @@ class SmartDropDownBox extends React.Component {
 
         this.state = {
             inputValue: '',
-            value: null,
+            //TODO default margin value should be zero
+            margin: this.props.margin != null ? this.props.margin : "0.5em",
+            options: this.props.options,
+            label: this.props.label,
+            value: this.props.preselectedValue
```

Comment:
fix the warning:
```suggestion
            value: this.props.preselectedValue ? this.props.preselectedValue : null,
```

maybe better to rename prop to `value`

Reply:
wieso die condition? wenn props leer sind haben sie eh den wert null